packages/ui/components/button/Button.tsx

```diff
@@ -190,7 +191,11 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
             <StartIcon
               className={classNames(
                 "inline-flex",
-                size === "icon" ? "h-4 w-4 " : "h-4 w-4 stroke-[1.5px] ltr:mr-2 rtl:ml-2"
+                size === "icon"
+                  ? "h-4 w-4 "
+                  : size === "iconsm"
```

Comment:
I find it hard to understand normally. Can we abstract it out into variables above this JSX.

Reply:
I would say yeah - but we  just loose the benefit of intelisense doing so